test/test_cp2k_parser.py

```diff
@@ -1,12 +1,13 @@
+"""Test CP2K components."""
+from pathlib import Path
 from qmflows.parsers import cp2KParser
-import os
 
 
 def test_read_cp2k_coefficients():
     """Test that the CP2K output coefficients are read properly."""
-    path = "test/test_files/output_cp2k"
-    mo_file = os.path.join(path, "mo_coeffs.log")
+    path = Path("test/test_files/output_cp2k")
```

Comment:
I'd recommend changing this into ``path = Path('test') / 'test_files' / 'output_cp2k')``; should cause less issues on different operating systems.

Reply:
There is also the constant (located in ``qmflows.test_utils.PATH``) which already points to ``"test/test_files"``.